src/v/config/property.h

```diff
@@ -31,10 +31,22 @@ class property : public base_property {
       config_store& conf,
       std::string_view name,
       std::string_view desc,
-      required req = required::yes,
+      base_property::metadata meta = {},
```

Comment:
Yep, that's intentional, to get the most common value as default.

There aren't any cluster config settings that are required, so in principle we could diverge node & cluster config properties and get rid of the field for cluster config, but for the present I wanted to keep them using a common structure in case we later decide to come back and do something like remote overrides of node config.